CHANGES_NEXT_RELEASE

```diff
@@ -1 +1,2 @@
 - Add: skipInitialNotification URI param to make initial notification in subscriptions configurable (#920)
+- Add: Forced Update option (#3389)
```

Comment:
I'd suggest:

```
Add: forcedUpdate URI param option to always trigger notifications, no matter if actual update or not (#3389)
```

(I have realized also that the line just above this one should read "URI param option" instead of just "URI param". Could you also fix it, please?)

Reply:
fixed in 1a4b51b